src/scenes/EndScene.ts

```diff
@@ -52,21 +54,25 @@ export class EndScene extends Scene<FriendlyFire> {
         }
     }
 
-    public draw(ctx: CanvasRenderingContext2D, width: number, height: number): void {
+    public draw(ctx: CanvasRenderingContext2D, size: Size): void {
         ctx.drawImage(
             EndScene.logo,
-            width / 2 - EndScene.logo.width / 2, height / 2 - EndScene.logo.height / 2 - 15
+            size.width / 2 - EndScene.logo.width / 2,
+            size.height / 2 - EndScene.logo.height / 2 - 15
         );
 
         if (this.time > this.subtitleDelay) {
             const endingLabel = this.ending ? this.ending.title : 'Unknown [E]nding';
-            const size = EndScene.font.measureText(endingLabel);
+            const endingLabelSize = EndScene.font.measureText(endingLabel);
 
             EndScene.font.drawText(
                 ctx,
                 endingLabel,
-                width / 2 - size.width / 2, height / 2 - EndScene.logo.height / 2 + 20,
-                "red"
+                new Point(
```

Comment:
Another two new Point instances per frame here. Again, not a crucial scene, but still...